### PR TITLE
Internal: [V2] Display control > Display Block [ED-15987]

### PR DESCRIPTION
**Pull request by ronenboxer**

### CODE REVIEW

File: modules/atomic-widgets/styles/style-schema.php [hunk 1]
```diff
@@ -152,4 +153,13 @@ private static function get_effects_props() {
 			'box-shadow' => Box_Shadow_Prop_Type::make(),
 		];
 	}
+
+	private static function get_layout_props() {
+		return [
+			'display' => String_Prop_Type::make()->enum([
+				'inline',
```
Comment (Omerisra6) [inline modules/atomic-widgets/styles/style-schema.php:160]: I think you should add all valid values here

/* precomposed values */
display: block;
display: inline;
display: inline-block;
display: flex;
display: inline-flex;
display: grid;
display: inline-grid;
display: flow-root;

/* box generation */
display: none;
display: contents;

**Review by Omerisra6 — CHANGES_REQUESTED** [review]

File: modules/atomic-widgets/styles/style-schema.php (hunk 1 above)
Comment (ronenelementor) [reply modules/atomic-widgets/styles/style-schema.php]: Done

File: modules/atomic-widgets/styles/style-schema.php [hunk 2]
```diff
@@ -152,4 +153,24 @@ private static function get_effects_props() {
 			'box-shadow' => Box_Shadow_Prop_Type::make(),
 		];
 	}
+
+	private static function get_layout_props() {
+		return [
+			'display' => String_Prop_Type::make()->enum([
+				/* precomposed values */
```
Comment (Omerisra6) [inline modules/atomic-widgets/styles/style-schema.php:160]:
```suggestion
```

File: modules/atomic-widgets/styles/style-schema.php [hunk 3]
```diff
@@ -152,4 +153,24 @@ private static function get_effects_props() {
...
+				'block',
+				'inline',
+				'inline-block',
+				'flex',
+				'inline-flex',
+				'grid',
+				'inline-grid',
+				'flow-root',
+
+				/* box generation */
```
Comment (Omerisra6) [inline modules/atomic-widgets/styles/style-schema.php:170]:
```suggestion
```

**Review by Omerisra6 — CHANGES_REQUESTED** [review]

File: modules/atomic-widgets/styles/style-schema.php (hunk 2 above)
Comment (ronenelementor) [reply modules/atomic-widgets/styles/style-schema.php:160]: Done

File: modules/atomic-widgets/styles/style-schema.php (hunk 3 above)
Comment (ronenelementor) [reply modules/atomic-widgets/styles/style-schema.php]: Done

**Review by Omerisra6 — APPROVED** [review]

**Pull request merged**